identifyConsensus.py

```diff
@@ -14,38 +14,40 @@ def get_parser():
                         help='List the network that corresponds to each results file (/directory if -d flag is set).')
     parser.add_argument('-d', '--delta_choice', action='store_true',
                         help='Automated delta selection. If flag is set, -r will expect a single directory per HotNet2 run.')
+    parser.add_argument('-t', '--p_value_threshold', type=float, default=0.01,
+                        help='Threshold for p-values; default is 0.01.')
     parser.add_argument('-o', '--output_file',required=True,
                         help='Output file. Text output by default. Use a .json extension to get JSON output.')
     parser.add_argument('-ms', '--min_cc_size', help='Min CC size.', type=int, default=2)
 
     return parser
 
-# Choose results files to use for consensus (1 per network)
-def choose_results( results_files, networks, min_cc_size, delta_choice ):
+# Choose results files to use for consensus (1 per network-score combination)
+def choose_results( results_files, min_cc_size, delta_choice, p_value_threshold ):
```

Comment:
Can we not pass in `delta_choice` here? Just don't call this method if `delta_choice` is `False`. Even better, have the `network_dirs` passed in as a separate parameter?